spec/controllers/admin/enterprise_fees_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: false
+
+require 'spec_helper'
+require 'open_food_network/order_cycle_permissions'
+
+RSpec.describe Admin::EnterpriseFeesController do
+  before { @request.env['HTTP_REFERER'] = 'http://test.com/' }
+
+  before do
+    allow(controller).to receive_messages spree_current_user: super_admin
+  end
+
+  describe "for_order_cycle" do
+    context "as super admin" do
+      let(:super_admin) { create(:admin_user) }
+      let!(:enterprise){ create(:distributor_enterprise_with_tax, name: 'Enterprise') }
+      let!(:fee1) { create(:enterprise_fee, :flat_rate, enterprise:) }
+      let!(:fee2) { create(:enterprise_fee, :per_item, enterprise:) }
+      let!(:fee3) { create(:enterprise_fee, :flat_rate, enterprise:) }
+      let!(:fee4) { create(:enterprise_fee, :per_item, enterprise:) }
+      let!(:order_cycle){
+        create(:simple_order_cycle, name: "oc1", suppliers: [enterprise],
+                                    distributors: [enterprise])
+      }
+
```

Comment:
We should cover the scenario when there is no `per_item` or `per_order` parameters.

Reply:
added